ide/refactoring.api/src/org/netbeans/modules/refactoring/spi/impl/RefactoringPanel.java

```diff
@@ -910,6 +926,7 @@ public void run() {
                         expandButton.setSelected(false);
                     }
                 } else {
+                    splitPane.setDividerLocation(0.3);
```

Comment:
`splitPane.setDividerLocation(0.3);` is now called from both branches -> can be moved outside of the if

Reply:
Done.